common/ruletypes.h

```diff
@@ -163,6 +163,7 @@ RULE_BOOL(Character, SoftDeletes, true, "When characters are deleted in characte
 RULE_INT(Character, DefaultGuild, 0, "If not 0, new characters placed into the guild # indicated")
 RULE_BOOL(Character, ProcessFearedProximity, false, "Processes proximity checks when feared")
 RULE_BOOL(Character, EnableCharacterEXPMods, false, "Enables character zone-based experience modifiers.")
+RULE_BOOL(Character, PVPEnableGuardFactionAssist, true, "Enables faction based assisting against the aggresor in pvp.")
```

Comment:
This should be default false is what splose is saying. I'm not sure actual behavior here.

Reply:
Project 1999 has guards that assist based on faction, I believe SZ and RZ both had this mechanic on live as well.

Reply:
I would have to agree `false` is better here.

Reply:
Live like from RZ suggests that this is proper default function